src/Microsoft.HttpRepl/UriLauncher.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Resources;
+
+namespace Microsoft.HttpRepl
+{
+    internal class UriLauncher : IUriLauncher
+    {
+        public Task LaunchUriAsync(Uri uri)
+        {
+            string agent = "cmd";
```

Comment:
This block reads a little weird (or very platform-biased) to me.  I think it would be more clear to break it down like

```
if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
{ /* ... */ }
else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
{ /* ... */ }
else // Linux
{ /* ... */ }
```

Reply:
Updated here:https://github.com/aspnet/HttpRepl/commit/a2823f1431f0069d0fc44df13bb8b9002a08198b
and here: https://github.com/aspnet/HttpRepl/commit/21d8173d634b7464bc65ebfffda0b44d4e720fb7